### ISSUE TITLE
How to train the maskgit transformers (base model)? 

### ISSUE THREAD

**Issue by easonoob** [lucidrains/muse-maskgit-pytorch#33]
How to train the maskgit transformers (base model)? 

The `trainers.py` is only for training the VQGAN-VAE model, and in readme, the code is just said getting the loss. There is no trainer for maskgit? How can I train it?
